operators/olm_v1/olmv1-managing-catalogs.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="olmv1-managing-catalogs"]
+= Managing catalogs for OLM v1 (Technology Preview)
+include::_attributes/common-attributes.adoc[]
+:context: olmv1-managing-catalogs
+
+toc::[]
+
+In OLM v1, a _plain bundle_ is a static collection of arbitrary Kubernetes manifests in YAML format. The experimental `olm.bundle.mediatype` property of the `olm.bundle` schema object differentiates a plain bundle (`plain+v0`) from a regular (`registry+v1`) bundle.
+
+:FeatureName: OLM v1
+include::snippets/technology-preview.adoc[]
+
+// For more information, see the [Plain Bundle Specification](https://github.com/operator-framework/rukpak/blob/main/docs/bundles/plain.md) in the RukPak repository.
+
+As a cluster administrator, you can build and publish a file-based catalog that includes a plain bundle image by completing the following procedures:
+
+. Build a plain bundle image.
+. Create a file-based catalog.
+. Add the plain bundle image to your file-based catalog.
+. Build your catalog as an image.
+. Publish your catalog image.
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../operators/olm_v1/olmv1-packaging-format.adoc#olmv1-packaging-format[RukPak component and packaging format]
+
+[id="prerequisites_olmv1-plain-bundles"]
+== Prerequisites
```

Comment:
A little confusing, since there are also prereqs (as `.Prerequisites`) in the procedure that follows. Does it make sense to roll them into the procedure prereqs? Could also see this as something like "requirements" or "preparing your environment".